crates/holochain/src/core/workflow/app_validation_workflow/tests.rs

```diff
@@ -507,6 +507,119 @@ async fn validate_ops_in_sequence_must_get_action() {
     assert_eq!(ops_to_validate, 0);
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn handle_error_in_op_validation() {
+    holochain_trace::test_run();
+
+    let entry_def = EntryDef::default_from_id("entry_def_id");
+    let zomes = SweetInlineZomes::new(vec![entry_def], 0).integrity_function(
+        "validate",
+        move |_, op: Op| match op {
+            Op::RegisterAgentActivity(_) => Err(InlineZomeError::TestError("kaputt".to_string())),
+            _ => Ok(ValidateCallbackResult::Valid),
+        },
+    );
+
+    let (dna_file, _, _) = SweetDnaFile::unique_from_inline_zomes(zomes).await;
+    let dna_hash = dna_file.dna_hash().clone();
+
+    let mut conductor = SweetConductor::from_config_rendezvous(
+        SweetConductorConfig::rendezvous(true),
```

Comment:
Why rendezvous with one conductor?

Reply:
I just always use rendezvous. Good point, I want to change that habit.

Reply:
done